pylib/mozhginfo/mozhginfo/push.py

```diff
@@ -0,0 +1,53 @@
+class Push(object):
+    """This represents a Push"""
+
+    def __init__(self, **kwargs):
+        """
+            We set Push object default as dict, and we will
+            give each push a push id when we init it.
+        """
+        self._id, self._push = kwargs.popitem()
```

Comment:
This is still not looking like this:
https://github.com/AutomatedTester/Bugsy/blob/master/bugsy/bug.py#L17